packages/test/test-utils/src/test/types/validateTestUtilsPrevious.generated.ts

```diff
@@ -199,6 +199,7 @@ declare function get_old_InterfaceDeclaration_ITestObjectProvider():
 declare function use_current_InterfaceDeclaration_ITestObjectProvider(
     use: TypeOnly<current.ITestObjectProvider>): void;
 use_current_InterfaceDeclaration_ITestObjectProvider(
+    // @ts-expect-error compatibility expected to be broken
```

Comment:
Is this ok in `main`?

Reply:
I think it should be fine since these changes are only to test packages. Also, the changes just make items that were already implemented by `TestObjectProvider` now required.